testing/scenariobuilder/validations.go

```diff
@@ -61,7 +61,7 @@ func validateJSONTodoErrs(programOutput string, scenarios []*TodoErrScenario) va
 		for _, elem := range elements {
 			removeIdx := -1
 			for j, expected := range scenarioJSONObjs {
-				if elem == *expected {
+				if elem.isEqual(expected) {
```

Comment:
a more idiomatic way for `isEqual` is `equals`

Reply:
`isEqual` -> `equals`